src/containers/Tablet/TabletInfo/TabletInfo.tsx

```diff
@@ -0,0 +1,77 @@
+import {Link} from 'react-router-dom';
+
+import {Link as UIKitLink} from '@gravity-ui/uikit';
+
+import {ETabletState, TTabletStateInfo} from '../../../types/api/tablet';
+import {InfoViewer, InfoViewerItem} from '../../../components/InfoViewer';
+import routes, {createHref} from '../../../routes';
+import {calcUptime} from '../../../utils';
+import {getDefaultNodePath} from '../../Node/NodePages';
+
+import {b} from '../Tablet';
+
+interface TabletInfoProps {
+    tablet: TTabletStateInfo;
+    tenantPath: string;
+}
+
+export const TabletInfo = ({tablet, tenantPath}: TabletInfoProps) => {
+    const {
+        ChangeTime,
+        Generation,
+        FollowerId,
+        NodeId,
+        HiveId,
+        State,
+        Type,
+        TenantId: {SchemeShard} = {},
+    } = tablet;
+
+    const tabletInfo: InfoViewerItem[] = [{label: 'Database', value: tenantPath}];
+
+    if (HiveId && HiveId !== '0') {
+        tabletInfo.push({
+            label: 'HiveId',
+            value: (
+                <UIKitLink href={createHref(routes.tablet, {id: HiveId})} target="_blank">
+                    {HiveId}
+                </UIKitLink>
+            ),
+        });
+    }
+
+    if (SchemeShard) {
+        tabletInfo.push({
+            label: 'SchemeShard',
+            value: (
+                <UIKitLink href={createHref(routes.tablet, {id: SchemeShard})} target="_blank">
+                    {SchemeShard}
+                </UIKitLink>
+            ),
+        });
+    }
+
+    tabletInfo.push({label: 'Type', value: Type}, {label: 'State', value: State});
+
+    if (State === ETabletState.Active) {
```

Comment:
it was a bit more readable with these conditions having names
`const hasUptime = State === ETabletState.Active;` would provide a little intel